sys-process/btop/btop-1.2.8.ebuild

```
@@ -13,6 +13,19 @@ LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="amd64 arm64 ppc64 ~riscv x86"
 
+BDEPEND="
+	>=sys-devel/gcc-8
+"
+
+pkg_setup() {
+	if [[ "${MERGE_TYPE}" != "binary" ]]; then
+		if [[ $(tc-getCXX) != *g++* ]] || [[ $(tc-getCXX) == *clang++* ]]; then
```

```suggestion
		if ! tc-is-gcc ; then
```

We require that CC/CXX and such are matching, so this isn't an issue.

---

thanks, I wasn't aware of that function